### DISCUSSION TITLE
Longhorn with spinning disks and software raid ?

### DISCUSSION THREAD

**Discussion by rsoika** [longhorn/longhorn#2495] [Q&A]
I have question about Longhorn running in a Kuberentes Cluster in combination with spinning discs. I know that SSD/NVMe is recommanded. But my question is:

Is it dangerous to use  2x4TB Spinning Disks with a Software Raid on Debian? I reccognized that I have kernel log messages like this one:

` kernel: [166395.795021] EXT4-fs error (device sdl): __ext4_find_entry:1449: inode #2: comm grafana-server: reading directory lblock 0`

Now I ask my self if I should switch to SSD?

**Comment by joshimoo** [comment]
We found some timeout issues when using spinning disks.
https://github.com/longhorn/longhorn/issues/2206

Which we plan on looking into and improve, unfortunately the improvement won't make the v1.1.1 release.

**Comment by rsoika** [comment]
Is it possible, that if I set the option "Data Locality" to "best-effort" the problem goes away?

I had a lot of disk trouble the last days with a new cluster using spinning disks. After I changed the "Data Locality" to "best-effort"  of most of my volumes it looks much better now. 

Can this be a workaround?
